lib/npm/test.js

```diff
@@ -28,6 +28,10 @@ function authorName(author) {
 }
 
 function test(context, next) {
+  if (context.options.script) {
+    // This handles the citgm.js case
+    context.module.script = context.options.script;
```

Comment:
Is this related to the script changes that just happened? Why is this line necessary